build.gradle.kts

```diff
@@ -205,20 +214,20 @@ subprojects {
     tasks.withType<JavaCompile>().configureEach { options.release.set(17) }
   }
 
-  val isSkatePlugin = project.path == ":skate-plugin"
+  val isIntelliJPlugin = project.hasProperty("INTELLIJ_PLUGIN")
   pluginManager.withPlugin("org.jetbrains.kotlin.jvm") {
     tasks.withType<KotlinCompile>().configureEach {
       compilerOptions {
         val kotlinVersion =
-          if (isSkatePlugin) {
+          if (isIntelliJPlugin) {
```

Comment:
Does intelliJ plugin right now include both the skate plugin and artifactory-authenticator?

Reply:
This is just for build logic, they're still both separate plugins